controller/argument_value_resolver.rst

```diff
@@ -146,10 +150,10 @@ and adding a priority.
 
         # app/config/services.yml
         services:
-            app.value_resolver.user:
-                class: AppBundle\ArgumentResolver\UserValueResolver
-                arguments:
-                    - '@security.token_storage'
+            # ...
+
+            AppBundle\ArgumentResolver\UserValueResolver:
+                # arguments is not needed if you have autowire above under _defaults
```

Comment:
This is the balance I'm thinking about a lot. We have some options:

1) Assume the user has the out-of-the-box `_defaults` settings, with `autowire` and `autoconfigure`. In that case, all code examples like this one are super simple

2) Don't assume the user has the correct `_defaults`, and show it at the top of every code block (or, add some comment like this one).

This is really a global decision to make - I'm already solving it slightly different in different places, as I'm not 100% sure.